drivers/SmartThings/matter-window-covering/src/init.lua

```diff
@@ -22,6 +22,9 @@ local MatterDriver = require "st.matter.driver"
 local DEFAULT_LEVEL = 0
 local PROFILE_MATCHED = "__profile_matched"
 
+local cap_call_dirty_bit = 0 -- BIT0: Current Position Attribute, BIT1:Operational Status Attribute
```

Comment:
Please replace this number with a constant that has a human-readable name explaining the meaning of the number.

Reply:
also it seems to be used as an enum, so it may be better to define the possible values.